L/libjulia/common.jl

```diff
@@ -15,6 +17,12 @@ function libjulia_platforms(julia_version)
         filter!(p -> !(Sys.islinux(p) && arch(p) == "powerpc64le"), platforms)
     end
 
+    # HACK: remove platforms that are not building with Julia 1.7.0-beta2 right now
+    if julia_version ≥ v"1.7"
+        filter!(p -> !(Sys.islinux(p) && arch(p) == "armv6l"), platforms)
+        filter!(p -> !(Sys.isapple(p) && arch(p) == "aarch64"), platforms)
```

Comment:
Is this still the case? :cry:

Reply:
This is because utf8proc is missing -- I think if someone updates https://github.com/JuliaBinaryWrappers/utf8proc_jll.jl/ to support these architectures, they might work (or at least progress a bit further). 

Actually, that should be mentioned in the comment above (I thought I had done so, but seems I forgot)

Reply:
macOS arm and Linux armv6l all fail with error `src/flisp/string.c:22:10: fatal error: utf8proc.h: No such file or directory`

Reply:
#3261